tests/exploreServicesBreakDown.spec.ts

```diff
@@ -403,19 +402,15 @@ test.describe('explore services breakdown page', () => {
     await explorePage.assertTabsNotLoading();
     // Fields on top should be loaded
     expect(requestCount).toEqual(6);
-
     await explorePage.scrollToBottom();
     // Panel on the bottom should be visible
     await expect(page.getByTestId(/data-testid Panel header/).last()).toBeInViewport();
-
     // Panel on the top should not
     await expect(page.getByTestId(/data-testid Panel header/).first()).not.toBeInViewport();
-
     // Wait for a bit for the requests to be made
     await page.waitForTimeout(250);
-
     // if this flakes we could just assert that it's greater then 3
-    expect(requestCount).toEqual(14);
+    expect(requestCount).toEqual(17);
```

Comment:
Don't know why there is an extra row now, this shouldn't have anything to do with the functional change in this PR, I think it's a consequence of the updates to 11.3